apps/avatax/src/modules/saleor/order.ts

```diff
@@ -47,3 +51,51 @@ export class SaleorOrder {
     return this.data.order.channel.taxConfiguration.pricesEnteredWithTax;
   }
 }
+export class SaleorCancelledOrder {
```

Comment:
**Suggestion:** what about having something like `Parser` and `SaleorCancelledOrder` instead? I think it will be easier to handle errors inside webhook handler instead in constructor 🤔

Reply:
Parser od factory method that will either create new instance of class or err()